### PR TITLE
Fixed Could not prevent wrong slider height when using image lazy load.

### PR DESCRIPTION
**Pull request by juanlopez4691**
Related to task https://app.asana.com/0/383242129844224/479915789522354

Other plugins using image lazyload techniques may cause the same issues solved for wp-rocket. Adding a filter allows to prevent this in a more general and flexible way.

### CODE REVIEW

File: src/Plugin.php [hunk 1]
```diff
@@ -130,7 +130,11 @@ public static function renderTemplate(array $template_subpathnames, array $varia
    * @return bool
    */
   public static function isLazyLoadActive() {
-    return is_plugin_active('wp-rocket/wp-rocket.php') && get_rocket_option('lazyload');
+    $lazyLoadIsActive = is_plugin_active('wp-rocket/wp-rocket.php') && get_rocket_option('lazyload');
+    if (has_filter('gallerya_lazyload_is_active')) {
```
Comment (LucaPipolo) [inline src/Plugin.php]: Just wondering if we need this conditional check before applying the filter. 🤔

Comment (juanlopez4691) [reply src/Plugin.php]: I'll remove it.

**Comment by juanlopez4691** [comment]
@sun This is not fixing any issue, but trying to solve a similar problem with bj-lazy-load plugin I realised that the fix for Gallerya slider display issues due to image lazy load was limited to wp-rocket plugin. Adding a filter provides additional flexibility to that fix.

**Comment by sun** [comment]
What's the actual fix then?

Why don't you simply enhance the condition and additionally check for whether the bj-lazy-load plugin is active?  Would that also resolve "the problem" (which hasn't been stated yet) without requiring additional adjustments anywhere else?

Please bear in mind that this is a public repository.

**Comment by juanlopez4691** [comment]
@sun I am afraid I have been not able to explain myself clearly, sorry for that.

1. This was done with the intention of fixing [this issue](https://app.asana.com/0/383242129844224/479915789522354) but it is not.
2. We are still searching a solution for that issue.
3. I left this PR opened because it still can be used for fixing the mentioned issue.
4. I decided to left the commit in this PR because IMO it is an improvement as it adds additional flexibility to the code.

Above all, my apologies for not labeling this as WIP, as I should have done from the begining.

**Pull request closed without merging**